model/labels/labels.go

```diff
@@ -455,12 +544,17 @@ Outer:
 	return b
 }
 
-// Set the name/value pair as a label.
+// Set the name/value pair as a label. A value of "" means delete that label.
```

Comment:
Why we need this "" semantics if we have `Del` method?

Reply:
Here I am documenting what the code did already.  I guess it is relied upon by relabling rules, though not explicitly in [the documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config).